pac4j-core/src/main/java/org/pac4j/core/client/Clients.java

```diff
@@ -179,14 +190,20 @@ public void setClients(final List<Client> clients) {
     public void setClients(final Client... clients) {
         this.clients = Arrays.asList(clients);
     }
-
+    
+    /**
+     * Returns initialized clients.
```

Comment:
In fact, this comment is misleading: getting clients via `getClients()` won't trigger any initialization while it will with the `findXX` methods (inline `init` call).